openmc/deplete/stepresult.py

```diff
@@ -0,0 +1,526 @@
+"""The results module.
```

Comment:
Minor confusion with `stepresults` saying  it's the results module

Reply:
Good catch -- I've updated this